postgresql-client/plan.sh

```diff
@@ -0,0 +1,20 @@
+source ../postgresql/plan.sh
+
+pkg_name=postgresql-client
+pkg_origin=core
+pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
+pkg_description="PostgreSQL is a powerful, open source object-relational database system."
+pkg_upstream_url="https://www.postgresql.org/"
+pkg_license=('PostgreSQL')
+pkg_filename="postgresql-${pkg_version}.tar.bz2"
+pkg_dirname="postgresql-${pkg_version}"
+
+# No exports/exposes for client
+unset pkg_exports
+unset pkg_exposes
+
+do_install() {
```

Comment:
I found it worthwhile to delete a bunch of files that are installed by default, but only used by postgresql plugins. This shrunk the installed size by 60%. See  https://github.com/habitat-sh/core-plans/blob/master/libpq/plan.sh#L81 for an example.

Reply:
great idea!